session/bootstrap.go

```diff
@@ -250,17 +251,26 @@ const (
 
 // bootstrap initiates system DB for a store.
 func bootstrap(s Session) {
-	b, err := checkBootstrapped(s)
-	if err != nil {
-		logutil.Logger(context.Background()).Fatal("check bootstrap error",
-			zap.Error(err))
-	}
-	if b {
-		upgrade(s)
-		return
+	dom := domain.GetDomain(s)
+	for {
+		b, err := checkBootstrapped(s)
+		if err != nil {
+			logutil.Logger(context.Background()).Fatal("check bootstrap error",
+				zap.Error(err))
+		}
+		// To reduce conflict when multiple TiDB-server start at the same time.
+		// Actually only one server need to do the bootstrap. So we chose DDL owner to do this.
+		if dom.DDL().OwnerManager().IsOwner() {
+			if b {
+				upgrade(s)
+				return
+			}
+			doDDLWorks(s)
+			doDMLWorks(s)
+			return
+		}
+		time.Sleep(50 * time.Millisecond)
```

Comment:
Is it possible to increase this value? Maybe we can set it to 200？

Reply:
done.